markdownPreview/src/index.tsx

```diff
@@ -15,14 +15,19 @@ import { createRoot } from "react-dom/client";
 import App from "./components/App";
 
 function init() {
-    const rootDiv = document.createElement("div", {});
-    rootDiv.setAttribute("id", "root");
-    const bodyInnerHTML = document.body.innerHTML;
-    const baseHref = (document.querySelector('base') || {}).href || "";
-    document.body.innerHTML = "";
-    document.body.appendChild(rootDiv);
+    const mdbd = document.getElementsByClassName("markdown-body");
+    const markdownBodyHTML = mdbd[mdbd.length - 1].outerHTML;
+    mdbd[mdbd.length - 1].remove();
```

Comment:
Waow ! Heavy stuff

Reply:
need to remove it so we have a clean html body :D